internal/app/skuba/cluster/init.go

```diff
@@ -83,6 +86,7 @@ func NewInitCmd() *cobra.Command {
 	if skuba.BuildType == "development" {
 		cmd.Flags().StringVar(&initOptions.KubernetesVersion, "kubernetes-version", "", "The kubernetes version to bootstrap with (only in development build)")
 	}
+	cmd.Flags().StringVar(&initOptions.CloudProvider, "cloud-provider", "", "Enable cloud provider integration with the chosen cloud. Valid values: openstack")
```

Comment:
Probably we would like to add `cloud-config` option with path to cloud provider configuration file, when it is provided by user or generated by terraform look on https://github.com/SUSE/skuba/pull/499

Reply:
I would try to not add the `--cloud-config` here until we have more information about how other cloud providers are integrated. `cloud.conf` is an OpenStack specific detail and I think we should try to avoid exposing specific workflow flags on the tool in something as generic as `skuba cluster init`.

The workflow would be that they run: `skuba cluster init --cloud-provider openstack`, they get generated the cluster definition folder with some [default `cloud.conf` as described in the RFC](https://github.com/SUSE/caasp-rfc/blob/master/accepted/0024-openstack-cloud-provider-integration.md), they change it as they want (they can copy some `cloud.conf` they have around and overwrite the sample one etc...), and then they bootstrap and join the nodes.